spec/features/reservation_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe "Reservation index list", type: :feature, js: true do
```

Comment:
I think that this test should go to unit test. Create 'unit' folder in spec and move the test there.